tests/base/src/test/resources/org/wildfly/security/http/client/wildfly-config.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just a minor, we should name this file wildfly-config-http-client-basic since we will add more files like this with different auth mechanisms to the tests later